cmd/podman/shared/funcs.go

```diff
@@ -6,24 +6,19 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/containers/libpod/pkg/util"
+	"github.com/containers/image/v5/types"
+	"github.com/containers/libpod/libpod/image"
 	"github.com/google/shlex"
+	"github.com/pkg/errors"
 )
 
-func GetAuthFile(authfile string) string {
```

Comment:
@TomSweeneyRedHat GetAuthFile has been removed here in cmd/podman/shared/funcs.go

Reply:
Ugh, sorry, must have been tired last night when reviewing.  I swore I searched fro funcs.go and didn't find any instances in the review.  Thanks.